media/js/base/mozilla-video-poster.js

```diff
@@ -4,7 +4,7 @@
 
 /* A simple HTML5 Video poster image helper.
  * https://bugzilla.mozilla.org/show_bug.cgi?id=1040213
- * CSS Styles: 'media/css/pebbles/_video-poster.scss' */
+ */
```

Comment:
Looks like there's no CSS for this now? I think that means we don't need to add/remove the `supports-video` class further down.

Reply:
This class still might be useful for pages that use the helper, so I think keeping it around won't do any harm